snapcraft/internal/meta/_snap_packaging.py

```diff
@@ -395,10 +399,27 @@ def _compose_snap_yaml(self):
             if key_name in self._config_data:
                 snap_yaml[key_name] = self._config_data[key_name]
 
+        passthrough_applied = False
+
         if 'apps' in self._config_data:
             _verify_app_paths(basedir='prime', apps=self._config_data['apps'])
             snap_yaml['apps'] = self._wrap_apps(self._config_data['apps'])
-            snap_yaml['apps'] = self._render_socket_modes(snap_yaml['apps'])
+            self._render_socket_modes(snap_yaml['apps'])
+            for app_name, app in snap_yaml['apps'].items():
+                if self._apply_passthrough(
+                        app_name, app, app.pop('passthrough', {}),
+                        self._original_snapcraft_yaml['apps'][app_name]):
+                    passthrough_applied = True
+
+        if self._apply_passthrough('snapcraft.yaml', snap_yaml,
+                                   self._config_data.get('passthrough', {}),
+                                   self._original_snapcraft_yaml):
+            passthrough_applied = True
+
+        if passthrough_applied:
+            logger.warn('Passthrough is being used to propagate experimental '
```

Comment:
Instead of Passthrough, I suggest using: `The 'passthrough' property is being used[...]`. That format matches up with some of our other warnings.